examples/tf/dask_script.py

```diff
@@ -0,0 +1,73 @@
+import click
+from dask.distributed import Client
+from dask_jobqueue import SLURMCluster
+
+from mnist_example import train_dense_model
+
+
+@click.command()
+@click.option(
+    'n_gpus',
+    '-n',
+    default=1,
+    help='The number of GPUs on which to run the mnist examples in parallel. Defaults to 1.',
+    type=int,
+)
+@click.option(
+    'save',
+    '-s',
+    '--save',
+    is_flag=True,
+    help='Whether you want to save the models or not',
+)
+def launch_dask_tasks(n_gpus, save):
+    assert 0 < n_gpus < 5, 'You need to request between 1 and 4 GPUs.'
+
+    job_name = 'dask_mnist_tf_example'
+    if n_gpus > 1:
+        job_name += '_multi_gpus'
+
+    cluster = SLURMCluster(
+        n_workers=n_gpus,
+        cores=n_gpus,
+        job_cpu=10,
+        memory='10GB',
+        job_name=job_name,
+        walltime='1:00:00',
+        interface='ib0',
+        job_extra=[
+            f'--gres=gpu:{n_gpus}',
+            '--qos=qos_gpu-dev',
+            '--distribution=block:block',
```

Comment:
I am curious about `distribution` and `nomultithread` below, have you ever tried to not use them and see what happens ?

My guess is that it should not make a huge difference but we copy and paste it from the Jean Zay documentation because we are not sure ...

Reply:
yes that is my feeling too.
I feel like these examples are a bit too simple to test that though, like some bad behaviours could happen when running longer.

Reply:
By the way I asked by email and they told me that `--distribution=block:block` can be removed now. On the other hand `--hint=nomultithread` is necessary ...